backend/internal/server/apis/schedule/pkg/csv/readCSV.go

```diff
@@ -0,0 +1,68 @@
+package csv
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"time"
+)
+
+type Schedule struct {
+	Date     time.Time
+	Time     time.Time
+	Duration time.Duration
+}
+
+//	 Reads CSV with the following format:
+//		[Date] [Time]
+//		YYYY-MM-DD, HH:MM:SS
+//		Returns an array of Schedule structs
+func readCSV(filename string) ([]Schedule, error) {
+	file, err := os.Open(filename)
+	if err != nil {
+		return nil, err
+	}
+	defer func(file *os.File) {
+		err := file.Close()
+		if err != nil {
+			fmt.Println("Error closing CSV file: ", err)
+			return
+		}
+	}(file)
+
+	reader := csv.NewReader(file)
+	lines, err := reader.ReadAll()
+	if err != nil {
+		return nil, err
+	}
+
+	var dateTimes []Schedule
+
+	for _, line := range lines {
+		dateStr := line[0]
+		timeStr := line[1]
+
+		date, err := time.Parse("2006-01-02", dateStr)
```

Comment:
Is that hardcoded date just for the layout?

Reply:
Just for layout